docker-entrypoint.sh

```diff
@@ -66,13 +66,25 @@ createSelfSignedServerCertificate() {
 }
 
 createTruststore() {
-    echo "Attempting to copy Java cacerts to create truststore."
-    $certificateManagerDir/certificate-manager.sh truststore --outputDirectory $securityDir --outputFile $APPLICATION_NAME.truststore
-    exitCode=$?
-    if [ ! $exitCode -eq 0 ];
+    if [ -f $dockerSecretDir/jssecacerts ];
     then
-        echo "Unable to create truststore (Code: $exitCode)."
-        exit $exitCode
+        echo "Custom jssecacerts file found."
+        echo "Copying file jssecacerts to the certificate location"
+        cp $dockerSecretDir/jssecacerts $securityDir/$APPLICATION_NAME.truststore
+    elif [ -f $dockerSecretDir/cacerts ];
```

Comment:
This is an else-if, so it would only be one (prioritizing the jssecacerts).

Reply:
Alert uses one blackduck-alert.truststore file when it starts up. so we have to pick one.  So we prioritize the same way the JDK does.  If there's a jssecacerts file we will use that for the trusted certificates.  Otherwise we will use cacerts if it exists.  We need to pick one file  because the application properties has the keystore and truststore file properties defined.